assets/_scss/components/_skipnav.scss

```diff
@@ -1,18 +1,19 @@
 .skipnav {
+  @include transition(top 1s ease-out, background 1s linear);
```

Comment:
@msecret I don't have a preference. It was just there before. Do you think we should remove it?